ANMN/LTSP/TSaggregator/aggregated_timeseries.py

```diff
@@ -361,7 +405,6 @@ def main_aggregator(files_to_agg, var_to_agg, site_code):
     site = 'NRSROT'
     files_to_aggregate = ['http://thredds.aodn.org.au/thredds/dodsC/IMOS/ANMN/NRS/NRSROT/Temperature/IMOS_ANMN-NRS_TZ_20141215T160000Z_NRSROT_FV01_NRSROT-1412-SBE39-33_END-20150331T063000Z_C-20180508T001839Z.nc',
     'http://thredds.aodn.org.au/thredds/dodsC/IMOS/ANMN/NRS/NRSROT/Temperature/IMOS_ANMN-NRS_TZ_20141216T080000Z_NRSROT_FV01_NRSROT-1412-SBE39-43_END-20150331T063000Z_C-20180508T001839Z.nc',
-    'http://thredds.aodn.org.au/thredds/dodsC/IMOS/ANMN/NRS/NRSROT/Biogeochem_timeseries/IMOS_ANMN-NRS_BCKOSTUZ_20151208T080040Z_NRSROT_FV01_NRSROT-1512-WQM-24_END-20160411T021734Z_C-20180504T071457Z.nc',
```

Comment:
Please remove all hard-coded URLs and "test" comments.

Reply:
removed.